packages/puppeteer-core/src/api/Browser.ts

```diff
@@ -441,5 +442,5 @@ export abstract class Browser extends EventEmitter<BrowserEvents> {
   /**
    * @internal
    */
-  abstract get protocol(): 'cdp' | 'webDriverBiDi';
+  abstract get protocol(): AutomationProtocol;
```

Comment:
How about just `Protocol`

Reply:
(I'm also implying we should renaming `Protocol` imports to something like `ProtocolTypes`)

Alternatively, `ProtocolName`

Reply:
Initially I did put it as just `Protocol` but it's rather obscure when seen next to other parts of the code. 
Let's discuss it offline how to proceed.